### ISSUE TITLE
spacing false positive

### ISSUE THREAD

**Issue by alok** [amperser/proselint#803]
I use pandoc to prettify markdown (via `pandoc -f markdown -t markdown`) and it puts 2 spaces after ordered lists to follow a 4 space indent rule. Proselint thinks this is an error. At least in the case where the ordered list is separated from the rest of the text by newlines, this may be worth detecting.

**Comment by compor** [comment]
I have the same issue. I use [remark-cli][1] to reformat markdown and I get a false positive for ordered list elements because they start with 2 spaces, e.g.

```markdown
1.  An item.
```
This triggers the `consistency.spacing` check.

I had a very brief look at the source code, and it seems that there is no notion of markdown constructs (or attempt to parse them). Am I correct? If this is true, will it require considerable effort to address?

[1]: https://github.com/remarkjs/remark
